sap/5.1/modules/ROOT/pages/sap-connector-config-snc.adoc

```diff
@@ -0,0 +1,171 @@
+
+* The SAP GUI application, which you can download from the https://sites.google.com/a/mulesoft.com/sap/abc/libs-tools?authuser=0[MuleSoft SAP Resource Center]
+* The SAPCAR utility and SAP Cryptographic Library, which you can download from the SAP Support Portal, if you are an S-user
+* An SAP ECC instance that can use SNC system variables
+* The following transactions:
+** `STRUST` (Trust Manager)
+** `SM30` (Table Maintenance)
+
+== Configure the Client Environment
+
+Configure the system variables that depend on the operating system you are using, and generate the personal security environment (PSE):
+
+. Open the Windows command line or Mac Terminal app.
+. Use the `SAPCAR` utility to unpack the SAP Cryptographic Library to the specified folder:
++
+`SAPCAR –xvf <folder-name.SAR>`
++
+. Configure the following environment variables:
+** SNC_LIB  `<path_to_library>\sapcrypto.dll`
+** SECUDIR = `<path_to_library>`
+** LD_LIBRARY_PATH= `<path_to_library>\sapcrypto.dll`
+** SSF_LIBRARY_PATH= `<path_to_library>\sapcrypto.dll`
+
++
+For Windows, the `sapcrypto.dll` 64-bit version must be in a different path and added to a new variable called `SNC_LIB_64`:
++
+image::sap-connector-windows-variable.png[Windows variable SNC_LIB_64]
++
+. Use the `sapgenpse` tool to create the PSE:
++
+`sapgenpse gen_pse -v -p RFC.pse`
++
+When you run this command, you must enter a PIN to use later and a distinguished name, which must be unique. For example, you can enter a distinguished name like this:
++
+`CN=RFC, C=AR, O=YourCompany, OU=IT`
++
+You can use STRUST to see names that are already in use.
++
+. Use the `sapgenpse` tool to create the `cred_v2` file, which stores the credentials for accessing the PSE:
++
+`sapgenpse seclogin -p RFC -O <user-credentials>`
+
++
+. Use the `sapgenpse` tool to export the client’s certificate from the PSE.
++
+`sapgenpse <export_own_cert> –v –p RFC –o RFC_MULE.crt`
+
+== Configure the SAP Server
+
+Configure the certificates and tables that authorize you to use SNC with your distinguished name. To configure the SAP server:
+
+. View the SAP distinguished name and certificates.
+. Link the SAP certificate to your certificate.
+. Authorize the SNC client in SAP.
+
+=== View the SAP Distinguished Name and Certificates
+
+View the SAP distinguished name and list of certificates:
+
+. Log in to the SAP GUI and enter the `strust` transaction.
+. Select the *SNC SAPCryptolib* folder in the left pane.
++
+The SAP distinguished name and certificate list are displayed.
+
+=== Link the SAP Certificate to Your Certificate
+
+Download the SAP instance certificate and link it to your certificate:
+
+. Click the glasses icon (first icon on the left of the screen) to enable the edition mode, and then double click `CN=SL-ABAP-IDP`.
+. Click the download button at the bottom of the screen and enter the target path for the download. *(Reviewers: is this text correct? Also, is the download a button or icon?)*
```

Comment:
It is a button, which is enable when you click on the glasses icon (edition mode)